compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -109,6 +111,8 @@ static void PrintMethodFields(
       // share the same request or response messages.
       p->Print(
```

Comment:
The TODO is still correct. There are two parsers being created for each method and there is no reuse when message types are shared.